components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/dao/impl/ApplicationDAOImpl.java

```diff
@@ -2649,14 +2649,8 @@ private AuthenticationScriptConfig getScriptConfiguration(int applicationId, Con
 
                     try {
                         boolean isEnabled = "1".equals(localAndOutboundConfigScriptResultSet.getString(2));
-                        StringBuilder sb = new StringBuilder();
-                        BufferedReader br = new BufferedReader(
-                                localAndOutboundConfigScriptResultSet.getCharacterStream(1));
-                        String line;
-                        while ((line = br.readLine()) != null) {
-                            sb.append(line).append("\r\n");
-                        }
-                        String targetString = sb.toString();
+                        String targetString = org.apache.commons.io.IOUtils.
```

Comment:
Why do we need to add full qualified class name here?

Reply:
We don't need it. I will fix it.